components/__tests__/locale-chooser.js

```diff
@@ -11,48 +11,111 @@ const {supportedLocales, fallbackLocale} = require('../../config.json')
 expect.addSnapshotSerializer(serializer)
 expect.extend(matcher)
 
-test('closed state', () => {
-  const wrapper = renderLocaleChooser()
+test('it renders', () => {
+  setHost()
+  const wrapper = mountLocaleChooser()
+  expect(toJson(wrapper)).toMatchSnapshotWithGlamor()
```

Comment:
I think maintaining this snapshot will just be really difficult. I would rather this test be simply:

```
test('it renders', () => {
  setHost()
  mountLocaleChooser()
})
```

Or just remove it entirely.